web/docs/_TypescriptServerNote.md

```diff
@@ -0,0 +1,11 @@
+:::caution LSP Problems
+
+If you are using TypeScript, your editor may sometimes report type and import errors even while `wasp start` is running.
+
+This happens when the TypeScript Language Server gets out of sync with the current code.
+If you're using VS Code, you can manually restart the language server by opening the command palette and selecting _"TypeScript: Restart TS Server."_
```

Comment:
```suggestion
If you're using VS Code, you can manually restart the language server by opening the command palette and selecting _TypeScript: Restart TS Server_.
```
I'd use only quotes or only italic styling.

Reply:
I like using both, especially for long text.

The quotes tell you when the quote start, and the cursive tells you that you're still inside the quote when reading it.

Are you ok with me keeping this style?